packages/application-components/src/components/internals/for-docs/example-wrapper/example-wrapper.js

```diff
@@ -159,6 +160,7 @@ const ExampleWrapper = props => {
                 title="Playground"
                 size="scale"
                 isOpen={isOpen}
+                zIndex={1001}
```

Comment:
This does not solve the issue.

![image](https://user-images.githubusercontent.com/1110551/59431623-9cd8ec80-8de5-11e9-8f74-bc617f03da25.png)

We need to set this higher, I would put it to `1100`, so like with an offset of `100` (just to be sure).

Reply:
Oh, I only tested with one opened Modal. 
👍